src/BlockTypes/ProductImage.php

```diff
@@ -115,21 +115,28 @@ private function render_on_sale_badge( $product, $attributes ) {
 	/**
 	 * Render anchor.
 	 *
-	 * @param \WC_Product $product Product object.
-	 * @param array       $attributes Attributes.
+	 * @param \WC_Product $product       Product object.
+	 * @param string      $on_sale_badge Return value from $render_image.
+	 * @param string      $product_image Return value from $render_on_sale_badge.
+	 * @param array       $attributes    Attributes.
 	 * @return string
 	 */
-	private function render_anchor( $product, $attributes ) {
+	private function render_anchor( $product, $on_sale_badge, $product_image, $attributes ) {
 		$product_permalink = $product->get_permalink();
 
-		$border_radius = StyleAttributesUtils::get_border_radius_class_and_style( $attributes );
```

Comment:
I also removed this `$border_radius` code from this `render_anchor` method as it does not seem to be used here.

It looks like it's handled / applied from the main `render` method below.

Unless I'm missing something?

Reply:
Good point! Thanks for catching this!